### PR TITLE
Add radiff2 -z test

### PR DESCRIPTION
**Pull request by l4l**
for radare/radare2#6452

### CODE REVIEW

File: t.tools/radiff2/radiff2 [hunk 1]
```diff
@@ -37,3 +37,15 @@ EXPECT='1
...
 
+
+NAME='radiff2 string comparsion'
+BROKEN=
+ARGS=
+CMDS="!!radiff2 -z ../../bins/elf/elf_one_symbol_shdr ./elf_one_symbol_shdr"
+EXPECT="0x000004b0 48656c6c => 01020304 0x000004b0
+"
+
+cp ../../bins/elf/elf_one_symbol_shdr ./elf_one_symbol_shdr
```
Comment (radare) [inline t.tools/radiff2/radiff2]: wtf

File: t.tools/radiff2/radiff2 [hunk 2]
```diff
@@ -37,3 +37,15 @@ EXPECT='1
...
+printf '\x01\x02\x03\x04' | dd of=elf_one_symbol_shdr bs=1 seek=1200 count=4 conv=notrunc
```
Comment (radare) [inline t.tools/radiff2/radiff2]: dont do that

Comment (radare) [inline t.tools/radiff2/radiff2]: use r2 to patch the binary if needed. but ideally the testsuite should run without modifying or copying files anywhere. you can maybe upload the same bin (if its small) twice with both changes to compare

**Comment by radare** [comment]
hello?

**Comment by l4l** [comment]
pong

**Pull request merged**